src/components/tutor-input.cjsx

```diff
@@ -31,6 +31,9 @@ TutorInput = React.createClass
     errors = @props.validate(@props.default)
     errors: errors or []
 
+  componentDidUpdate: (prevProps, prevState) ->
+    @props.onUpdated?(@state) unless _.isEqual(prevState, @state)
+
```

Comment:
oh, yah :( good one! teehehehe i tend to neglect the `propTypes` and i've notice you always do a good job with them